cmd/minishift/cmd/start.go

```diff
@@ -324,7 +324,7 @@ func startHost(libMachineClient *libmachine.Client) *host.Host {
 	isRestart := cmdutil.VMExists(libMachineClient, constants.MachineName)
 	if !isRestart {
 		fmt.Println("-- Minishift VM will be configured with ...")
-		fmt.Println("   Memory:   ", units.HumanSize(float64((machineConfig.Memory/units.KiB)*units.GB)))
+		fmt.Println("   Memory:   ", units.BytesSize(float64(machineConfig.Memory*units.MiB)))
```

Comment:
I really dislike that there is a seemingly change of unit. I specify --memory 5GB on the CLI and the I get displayed "    Memory : 5GiB". 

I see several options here:

* In the print statement we actually use the value as given by Viper, so this way 5GB stays 5GB, etc
* We enforce the use of MiB or GiB. GB and MB would not be allowed in this case
* We always convert to MiB for displaying the actual configured memory